app/src/ui/notification/new-commits-banner.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react'
+import { ButtonGroup } from '../lib/button-group'
+import { Button } from '../lib/button'
+import { Ref } from '../lib/ref'
+import { Octicon, OcticonSymbol } from '../octicons'
+import { Branch } from '../../models/branch'
+
+interface INewCommitsBannerProps {
+  /**
+   * The number of commits behind `branch`
```

Comment:
I tried to make the prop name as self-documenting as possible, so this could be removed, but I'll leave that to whoever reviews this.

Reply:
:fire::fire::fire: if you can make `commitsBehind` a bit more contextual - commits behind _what_?